.pre-commit-config.yaml

```
@@ -23,7 +23,8 @@ repos:
   rev: v1.29.0
   hooks:
     - id: yamllint
-      args: [-f=github, -c=.yamllint.yml, tests/regression/tests]
+      args: [-f=github, -c=.yamllint.yml]
+      files: '^tests/regression/tests/'
```

We have other YAML files. How about this?
```suggestion
      files: '\.ya?ml'
```